ArcadiaParties-SPA/package.json

```diff
@@ -12,10 +12,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~9.0.1",
+    "@angular/cdk": "^9.0.1",
```

Comment:
Do you use it somewhere?

Reply:
It was automatically added after angular-material installation. Should I delete this line?

Reply:
How did you add angular-material? If you run `npm i @angular/material -S` it should not had added another packages.

Reply:
I used `ng add @angular/material` like in official guide. 

> The ng add command will install Angular Material, the Component Dev Kit (CDK), Angular Animations

Reply:
Ok, now you can remove CDK dependency from the project, you won't need it at all.

Reply:
Ok, after some investigation I realized that I was wrong and this project is needed for @angular/material correct work.

Reply:
@angular/material and @angular/cdk returned.